RobotLib/src/main/java/com/acmerobotics/library/dashboard/RobotDashboard.java

```diff
@@ -69,9 +70,7 @@ public boolean shouldProcessClass(String className) {
             public void processClass(Class clazz) {
                 if (clazz.isAnnotationPresent(Config.class)) {
                 	Log.i(TAG, String.format("Found config class %s", clazz.getCanonicalName()));
-                    Config annotation = (Config) clazz.getAnnotation(Config.class);
-                    String name = annotation.value().equals("") ? clazz.getSimpleName() : annotation.value();
-                    optionGroups.add(new OptionGroup(clazz, name, prefs));
```

Comment:
what are going on with these variable names?

Reply:
Yeah it's weird... I'll change them all to `klass`